lib/std/builtin.zig

```diff
@@ -462,6 +462,13 @@ pub const WasiExecModel = enum {
     reactor,
 };
 
+/// This data structure is used by the Zig language code generation and
+/// therefore must be kept in sync with the compiler implementation.
+pub const DwarfFormat = enum {
+    dwarf32,
+    dwarf64,
+};
+
```

Comment:
Why is this in std.builtin? Seems like it belongs in `std.dwarf`, no?

Reply:
Oh I didn't know where to put it, but yeah, it makes sense. Fixed.